test/integration/targets/cloudwatchlogs/playbooks/roles/cloudwatchlogs_log_group_metric_filter/tasks/main.yml

```diff
@@ -0,0 +1,151 @@
+---
+- name: set up aws connection info
```

Comment:
If you use the module_default block below you can remove this and all of the ```<<: *aws_connection_info``` entries